builder/dockerfile/builder.go

```diff
@@ -253,6 +253,10 @@ func (b *Builder) build(stdout io.Writer, stderr io.Writer, out io.Writer) (stri
 			// Not cancelled yet, keep going...
 		}
 
+		if strings.EqualFold("FROM", n.Value) && b.options.Target != "" && strings.EqualFold(b.imageContexts.currentTargetName(), b.options.Target) {
```

Comment:
Can we use the `command.From` constant here? I believe the `n.Value` is already normalzied to lowercase.

It would be nice to have dispatch support a return value that would be used for exiting the build, but at this point I guess it isn't necessary yet.